lib/Db/Alias.php

```diff
@@ -63,6 +68,7 @@ public function jsonSerialize(): array {
 			'name' => $this->getName(),
 			'alias' => $this->getAlias(),
 			'signature' => $this->getSignature(),
+			'provisioned' => (bool)$this->getProvisioningId(),
```

Comment:
```suggestion
			'provisioned' => $this->getProvisioningId() !== null,
```

maybe the bool cast works too but :see_no_evil: